jenkins-pipelines/longevity-twcs-bench-3h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_twcs_bench_test.TWCSLongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-twcs-bench-3h.yaml',
+
+    timeout: [time: 300, unit: 'MINUTES']
```

Comment:
no need to set timeout, it is set in `'Get test duration'` stage using `getJobTimeouts` function (base value that is used it the test_duration from the test config YAMLs)

Reply:
@bentsi when removing this property got error
[2021-02-10T13:32:12.840Z] groovy.lang.MissingPropertyException: No such property: testRunTimeout for class: longevityPipeline